### PR TITLE
HW 10 and sample place

### PR DESCRIPTION
**Pull request by koalaboy808**
still need to clean up seventeen2... but runs appropriately (I think) right now

### CODE REVIEW

File: number_text.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def number_text(text):	
+
+	with open(text) as f_in:
```
Comment (danielsgriffin) [inline number_text.py]: rather than a counter, you could try enumerate()

File: seventeen1.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			
+
+# function for user input
+def human_turn(marbles):	
+	while True:
+		marbles_taken = raw_input("Your turn: How many marbles will you remove (1-3)?")
+		# use try/except to make sure the number is a digit
+		try:
+			int(marbles_taken)
+		except:
```
Comment (danielsgriffin) [inline seventeen1.py]: We want to start catching specific errors:
```python
except ValueError:
```

File: seventeen1.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# Instructions
+'''
+
+seventeen1
+
+'''
+
+# import
+
+import sys
```
Comment (danielsgriffin) [inline seventeen1.py]: you use this?

File: seventeen1.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+			print "There are no marbles left"
+			print winner
+			break
+		# Computer
+		elif (turn % 2)==0:
+			turn += 1
+			winner = "You win!"
+			marbles = computer_turn(marbles)
+		# Human
+		elif (turn % 2)==1:
```
Comment (danielsgriffin) [inline seventeen1.py]: does this need to be an elif?

File: seventeen2.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+		f_in_list = []
+		f_in = f_in.read().split("\n")
+		# loop through list of strings and split into lists within lists by ","
+		[f_in_list.append(line.split(",")) for line in f_in]
+		return f_in_list
+
+def new_file():
+	f_in_list = read_file()
+
+	with open("seventeen_output.txt", "w") as f_out:
```
Comment (danielsgriffin) [inline seventeen2.py]: That's a lot to run while the file is open. Depending on the situation may/mayn't be a good idea.

Comment (danielsgriffin) [inline seventeen2.py]: It also doesn't make it clear to someone looking at your code what you are writing

File: seventeen2.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+					f_out.write(". Winner: P1")
+					computer_win += 1
+					break
+		f_out.write("\nPlayer 1 won " + str(computer_win) + "times; Player 2 won " + str(human_win) + " times.")
+
+
+#############################################################################
+##################### call functions below through main(): ###################
+##############################################################################
+def main():
```
Comment (danielsgriffin) [inline seventeen2.py]: Good job.

**Comment by danielsgriffin** [comment]
Good job.

File: seventeen2.py (hunk 6 above)
Comment (danielsgriffin) [inline seventeen2.py]: note this: "Player 1 won 1times; Player 2 won 9 times."
1times

File: number_text.py (hunk 1 above)
Comment (koalaboy808) [inline number_text.py]: o ya... that would be cleaner

File: seventeen1.py (hunk 3 above)
Comment (koalaboy808) [inline seventeen1.py]: no... should be deleted. i mis-read the instructions and thought we had to write out everything from the command line to a text file. found a tricky way using sys

File: seventeen1.py (hunk 4 above)
Comment (koalaboy808) [inline seventeen1.py]: no it doesn't... but for my own readability I wanted to know what I was specifically testing

File: seventeen1.py (hunk 2 above)
Comment (koalaboy808) [inline seventeen1.py]: right... I need to do reading on error stuff. one of the things I don't feel comfortable with yet

File: seventeen1.py (hunk 4 above)
Comment (danielsgriffin) [inline seventeen1.py]: there is a performance cost though (eventually).
instead, perhaps:
else:  # turn % 2 == 1

**Pull request closed without merging**